code/game/machinery/bots/ed209bot.dm

```diff
@@ -1022,9 +1022,11 @@ Auto Patrol: []"},
 				var/turf/T = get_turf(user)
 				to_chat(user, "<span class='notice'>You start to wire [src]...</span>")
 				sleep(40)
```

Comment:
These should actually be if(do_after(user, src, 40) && build_step == 6) and then tab everything below it forward (and remove the get_turf(user) == T, this is just a poor version of do_after)